README.md

```diff
@@ -139,24 +141,24 @@ Esri welcomes contributions from anyone and everyone. Please see our [guidelines
 
 ## Terms and Conditions
 
-In order to make use of the ArcGIS Online World Geocoding Service in your web application:
+In order use the ArcGIS geocoding service you need the following:
+
+1. An [ArcGIS Developer account](https://developers.arcgis.com/sign-up/) or [ArcGIS Online account](https://www.esri.com/en-us/arcgis/products/arcgis-online/trial).
+2. If your application uses an basemap layer service or other Esri data, you must display`Powered by`[`Esri`](http://esri.com) in the map attribution. See [Basemap attribution](https://developers.arcgis.com/documentation/mapping-apis-and-services/deployment/basemap-attribution/) for more details.
 
-1. Please sign up for an [ArcGIS for Developers account](https://developers.arcgis.com/en/plans) or purchase an [ArcGIS Online Organizational Subscription](http://www.arcgis.com/features/plans/pricing.html).
-2. Ensure that `Powered by`[`Esri`](http://esri.com) is displayed in the map attribution.
+## Not-stored vs stored geocodes
 
-## Cost
+To determine if there is a cost associated with using the Esri Leaflet Geocoder and the ArcGIS geocoding service, if you have an ArcGIS Developer account, go to [Pricing](https://developers.arcgis.com/pricing/). If you have an ArcGIS Online account, go to [Credits by Capability](https://doc.arcgis.com/en/arcgis-online/administer/credits.htm#GUID-D309A4D0-43CD-4E58-A6DF-012A82A6D794).
```

Comment:
```suggestion
To determine if there is a cost associated with using the Esri Leaflet Geocoder and the ArcGIS geocoding service, if you have an ArcGIS Developer account, go to [Pricing](https://developers.arcgis.com/pricing/), or, if you have an ArcGIS Online account, go to [Credits by Capability](https://doc.arcgis.com/en/arcgis-online/administer/credits.htm#GUID-D309A4D0-43CD-4E58-A6DF-012A82A6D794).
```

Reply:
@alaframboise All suggestions applied, but there was one grammatical issue with the if's, this may work better if we break it into a bulleted list for the 2 different options

Reply:
I'll merge this change and we can switch it to bulleted list in a follow-up if necessary. Thanks!